apps/example/screens/ShieldTab.tsx

```
@@ -0,0 +1,324 @@
+            }, 100);
+          }}
+        />
+        <Text>
+          {familyActivitySelectionResult &&
+          familyActivitySelectionResult?.categoryCount < 13
+            ? `${familyActivitySelectionResult?.applicationCount} apps, ${familyActivitySelectionResult?.categoryCount} categories, ${familyActivitySelectionResult?.webDomainCount} domains`
+            : familyActivitySelectionResult?.categoryCount
+              ? "All categories selected"
+              : "Nothing selected"}
+        </Text>
+        <Text>{"intersection: " + JSON.stringify(intersection, null, 2)}</Text>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)